tests/utils/test_time_utils.py

```diff
@@ -23,10 +23,13 @@ def test_current_datetime(self):
 
         now = datetime.datetime.now(pytz.timezone("Etc/GMT-0"))
         test_time = time_utils.current_datetime()
+        time_delta = datetime.timedelta(milliseconds=100)
+        before = now - time_delta
+        after = now + time_delta
 
         assert (
-            now < test_time < (now + datetime.timedelta(milliseconds=100))
-        ), f"Unexpected time value {test_time} should be close to {now}"
+            before < test_time < after
```

Comment:
This is testing something different now, right? Why did you make the change?

Reply:
now < time_time seemed like a spurious test to me anyway, since both are initialized the exact same way